### PR TITLE
Control center settings tab 

### PR DESCRIPTION
**Pull request by BananiMendix**
New settings tab in control center data Hub administration page and default discoverability status setting

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
Thanks @BananiMendix is this in production already? Is it GA?

**Comment by BananiMendix** [comment]
> Thanks @BananiMendix is this in production already? Is it GA?

Not in production yet. We are planning to deploy it this week.

**Comment by Adam-Dupaski** [comment]
@BananiMendix when this week, and will there be release notes for this?

File: content/data-hub/data-hub-catalog/register-data.md [hunk 1]
```diff
@@ -353,3 +353,5 @@ Follow the steps below:
 8. Select the **Done!** option that appears once you have filled out all the required fields.
 
 Congratulations! Your OData service is registered in the Data Hub Catalog. 
+
+The discoverable status of the OData service defaults to the value set by the company admin. To learn more about it [click here](#datasource-settings)
```
Comment (Adam-Dupaski) [inline content/data-hub/data-hub-catalog/register-data.md]: @BananiMendix what doc/section does this cross-reference direct to? There is no anchor set with this name.

File: content/data-hub/data-hub-catalog/register-data.md [hunk 2]
```diff
@@ -353,3 +353,5 @@ Follow the steps below:
 8. Select the **Done!** option that appears once you have filled out all the required fields.
 
 Congratulations! Your OData service is registered in the Data Hub Catalog. 
+
+The discoverable status of the OData service defaults to the value set by the Mendix Admin. For more details, see [Data Source Settings](#datasource-settings).
```
Comment (Adam-Dupaski) [inline content/data-hub/data-hub-catalog/register-data.md:357]: @BananiMendix what doc/section does this cross-reference direct to? There is no anchor set with this name.

**Comment by BananiMendix** [comment]
> @BananiMendix when this week, and will there be release notes for this?

@Adam-Dupaski - HI Adam , we will postpone this deployment to next week(2 November,2021) , yes there will be release note

**Comment by dtitenko** [comment]
Superseded by https://github.com/mendix/docs/pull/3927

**Pull request closed without merging**